python/pynessie/types.py

```diff
@@ -35,7 +35,7 @@ def convert(self, value: Any, param: Optional[Parameter], ctx: Optional[Context]
         try:
             return ContentKey.from_path_string(str(value))
         except (TypeError, ValueError) as e:
-            self.fail(f"{value!r} is not a valid content key due to: '{e}'", param, ctx)
+            self.fail(f"{value!r} is not a valid content key due to: '{e!r}'", param, ctx)
```

Comment:
nit: if we use the `repr(..)` of the exception, putting it in quotes is redundant as it would look like this `due to: 'TypeError("foobar")'`

for `B028` on https://pypi.org/project/flake8-bugbear/ they suggest removing the quotes when using `!r`